examples/swarmion-with-next/frontend/next-app/package.json

```diff
@@ -3,14 +3,17 @@
   "private": true,
   "version": "1.0.0",
   "scripts": {
+    "bootstrap": "cdk bootstrap",
```

Comment:
can you ensure that nx runs bootstrap before deploy?
there is an example [here](https://github.com/swarmion/swarmion/pull/351)

Reply:
Yes, I added the nx configuration to cache and run the command!